crates/rust-analyzer/tests/slow-tests/support.rs

```diff
@@ -71,7 +71,11 @@ impl<'a> Project<'a> {
         let tmp_dir = self.tmp_dir.unwrap_or_else(TestDir::new);
         static INIT: Once = Once::new();
         INIT.call_once(|| {
-            env_logger::builder().is_test(true).parse_env("RA_LOG").try_init().unwrap();
+            tracing_subscriber::fmt()
+                .with_test_writer()
+                .with_env_filter(tracing_subscriber::EnvFilter::from_env("RA_LOG"))
+                .try_init()
```

Comment:
I think you can call `.init()` instead of `.try_init().unwrap()`.

Reply:
ok